packages/vant/src/picker/Picker.tsx

```diff
@@ -159,11 +159,16 @@ export default defineComponent({
         // reset values after cascading
         selectedValues.value.forEach((value, index) => {
           const options = currentColumns.value[index];
-          if (!isOptionExist(options, value, fields.value)) {
-            setValue(
-              index,
-              options.length ? options[0][fields.value.value] : undefined,
-            );
+          // reset when the first argument is changed
```

Comment:
Without context, I'm having trouble understanding the `argument` here. Please either remove this comment or provide sufficient explanation.